ark/segmentation/marker_quantification_test.py

```diff
@@ -124,48 +130,48 @@ def test_compute_marker_counts():
 
 
 def test_generate_expression_matrix():
-    cell_mask, channel_data = _create_test_extraction_data()
+
+    cell_mask, channel_data = test_utils.create_test_extraction_data()
 
     # generate data for two fovs offset
     cell_masks = np.zeros((2, 40, 40, 1), dtype="int16")
-    cell_masks[0, :, :, 0] = cell_mask
-    cell_masks[1, 5:, 5:, 0] = cell_mask[:-5, :-5]
+    cell_masks[0, :, :, 0] = cell_mask[0, :, :, 0]
+    cell_masks[1, 5:, 5:, 0] = cell_mask[0, :-5, :-5, 0]
 
-    channel_datas = np.zeros((2, 40, 40, 5), dtype="int16")
-    channel_datas[0, :, :, :] = channel_data
-    channel_datas[1, 5:, 5:, :] = channel_data[:-5, :-5]
+    tif_data = np.zeros((2, 40, 40, 5), dtype="int16")
+    tif_data[0, :, :, :] = channel_data[0, :, :, :]
+    tif_data[1, 5:, 5:, :] = channel_data[0, :-5, :-5, :]
 
-    segmentation_masks = xr.DataArray(cell_masks,
-                                      coords=[["Point1", "Point2"], range(40), range(40),
-                                              ["whole_cell"]],
-                                      dims=["fovs", "rows", "cols", "compartments"])
+    segmentation_masks = test_utils.make_labels_xarray(
+        label_data=cell_masks,
+        compartment_names=['whole_cell']
+    )
 
-    channel_data = xr.DataArray(channel_datas,
-                                coords=[["Point1", "Point2"], range(40), range(40),
-                                        ["chan0", "chan1", "chan2", "chan3", "chan4"]],
-                                dims=["fovs", "rows", "cols", "channels"])
+    channel_data = test_utils.make_images_xarray(
+        tif_data=tif_data
+    )
 
-    normalized, arcsinh = marker_quantification.generate_expression_matrix(segmentation_masks,
-                                                                           channel_data)
+    normalized, _ = marker_quantification.generate_expression_matrix(segmentation_masks,
+                                                                     channel_data)
 
     assert normalized.shape[0] == 7
 
-    assert np.all(normalized['chan0'] == np.repeat(1, len(normalized)))
-    assert np.all(normalized['chan1'] == np.repeat(5, len(normalized)))
-    assert np.all(normalized['chan2'] == normalized['chan2'])
```

Comment:
You removed our most crucial test

Reply:
issue 5049102: consolidate all lingual tests of equality into a `test_equalities.py` file. some examples:
 - 1 == 1
 - 2 == 2
 - n == n (we might be able to generalize this sequence by using 20 different scipy packages and machine learning)
 - `normalized['chan2'] == normalized['chan2']`
 - `a_list == [i for i in a_list if i == i]`
 - `False == True` (have we verified this?)
 - `the_joke == is_going_on_a_bit_long_dont_you_think`